04_03_defuse_0_6_2_Dockerfile

```diff
@@ -0,0 +1,29 @@
+# ProTECT Dockerfile for deFuse 0.6.2
+#
+# Build with: sudo docker build --force-rm --no-cache -t aarjunrao/defuse:0.6.2 - < 04_03_defuse_4_3_Dockerfile
+# Run with sudo docker run -v <IO_folder>:/data aarjunrao/defuse:0.6.2 <parameters>
+
+# Use pimmuno
+FROM aarjunrao/pimmuno:pimmuno
+
+MAINTAINER Jacob Pfeil, jpfeil@ucsc.edu
+
+RUN echo 'deb http://us.archive.ubuntu.com/ubuntu trusty main multiverse' >> /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y libboost-all-dev \
+                                         samtools \
+                                         bowtie \
+                                         gmap \
+                                         realpath \
+                                         r-base-dev
+
+RUN R -e 'install.packages("ada", repos="http://cran.cnr.berkeley.edu")'
+
+
+WORKDIR /home
+
+RUN wget -P /user/local/bin http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v287/faToTwoBit && \
+    wget -P /user/local/bin http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v287/blat/blat && \
+    chmod +x /user/local/bin/faToTwoBit /user/local/bin/blat
+
+RUN wget http://downloads.sourceforge.net/project/defuse/defuse/0.6/defuse-0.6.2.tar.gz && tar xf defuse-0.6.2.tar.gz
```

Comment:
Need to add in the code for a wrapper script (look at any other file in this repo) and then make a wrapper which will be deposited at https://github.com/arkal/toil_docker_wrappers